packages/webapp/src/components/SettingsStyles/SettingsStyles.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState, useEffect } from 'react';
+import './SettingsStyles.css';
+
+export type CustomStylesState = {
+  AccentHigh: string;
+  AccentLight: string;
+  AccentDark: string;
+  TextColor: string;
+  BackgroundColor: string;
+  FormBackgroundColor: string;
+  BorderRadius: string;
+  BorderWidth: string;
+  FontBaseSize: string;
+};
+
+interface Props {
+  onChange: (newStyles: CustomStylesState) => void;
+  isDarkTheme: boolean;
+}
+
+export const SettingsStyles = ({ onChange, isDarkTheme }: Props) => {
+  // this needs to come from an API call to some config persisted in the DB
+  const styleDefaultsLight = {
+    AccentHighDefault: '#692b61',
+    AccentLightDefault: '#f6d5f2',
+    AccentDarkDefault: '#5e3c7d',
+    TextColorDefault: '#123f58',
+    BackgroundColorDefault: '#e3e3e3',
+    ForegroundColorDefault: '#4e5288',
+    FormBackgroundColorDefault: '#f5f5f5',
+    BorderRadiusDefault: '10px',
+    BorderWidthDefault: '3px',
+    FontBaseSizeDefault: '14px',
+  };
+
+  const styleDefaultsDark = {
+    AccentHighDefault: '#dcdef8',
+    AccentLightDefault: '#032219',
+    AccentDarkDefault: '#fdfeff',
+    TextColorDefault: '#fdfeff',
+    BackgroundColorDefault: '#e3e3e3',
+    ForegroundColorDefault: '#4e5288',
+    FormBackgroundColorDefault: '#32343e',
+    BorderRadiusDefault: '10px',
+    BorderWidthDefault: '3px',
+    FontBaseSizeDefault: '14px',
+  };
```

Comment:
we should probably have a themes folder with different themes in a different file that can be imported in where these live rather than have them here.

Reply:
Yes. I will refactor that.